index.js

```diff
@@ -980,6 +980,54 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] | String -> Maybe [a] | Maybe String
```

Comment:
We could pretend that JavaScript has a Char type and that String is sugar for [Char]. Then we could write:

```javascript
//# take :: Number -> [a] -> Maybe [a]
```

If one substitutes `Char` for `a` then `String` for `[Char]` one gets `Number -> String -> Maybe String`.